ethpm/package.py

```diff
@@ -190,12 +190,13 @@ def build_dependencies(self) -> "Dependencies":
     #
 
     @cached_property
-    def deployments(self) -> "Deployments":
+    def deployments(self) -> Union["Deployments", Dict[None, None]]:
```

Comment:
The `Dict[None, None]` is confusing... I think it's intended to represent returning an empty dictionary.  Maybe it'd be better to have the return value be `None` in that case, though that does mean you have to do return value checking which isn't ideal.

Reply:
Yup, leaning towards just leaving the confusing type in for now to make life slightly easier in `pytest-ethereum` re. return value checking